### PR TITLE
test correct usage of tokens

### PR DESCRIPTION
**Pull request by individual-it**
a couple of security relevant tests, more to follow

### CODE REVIEW

File: tests/acceptance/features/webUIOauth2/obtainAccessToken.feature [hunk 1]
```diff
@@ -16,16 +16,21 @@ Feature: obtaining an access token
...
 		And the user authorizes the oauth app using the webUI
 		And the client app requests an access token
 		Then the client app should be able to download the file "lorem.txt" of "user1" using the access token for authentication
+		But the client app should not be able to download the file "lorem.txt" of "user2" using the access token for authentication
 
 	Scenario: receive a new access token by using the refresh token
 		Given the user "user1" has correctly established an oauth session
 		When the client app refreshes the access token
 		Then the client app should be able to download the file "lorem.txt" of "user1" using the access token for authentication
+		But the client app should not be able to download the file "lorem.txt" of "user2" using the access token for authentication
```
Comment (phil-davis) [inline tests/acceptance/features/webUIOauth2/obtainAccessToken.feature]: ``user2`` does not exist in this scenario, what is the intended test here?

Comment (individual-it) [reply tests/acceptance/features/webUIOauth2/obtainAccessToken.feature]: thank you for the tip, corrected

**Comment by patrickjahns** [comment]
I am curious - have you done a reverse test (i.e. allow refresh even though it shouldn't be allowed ) and ensured that the tests catch the assumed error?

**Comment by individual-it** [comment]
I'm currently thinking of different things to be tested in oauth, so if you have ideas please tell me. I think I've covered most things in the test-plan https://github.com/owncloud/QA/blob/master/Server/Test_Plan_OAuth2.md (need to update it) but more scenarios need to be covered

**Pull request merged**